galaxy/importer/linters/yamllint.yaml

```diff
@@ -0,0 +1,19 @@
+extends: default
```

Comment:
Would it be useful to add a comment here explaining it's based on the ansible-lint rules.
(+1 for mentioning it in the commit message).